src/index.d.ts

```diff
@@ -1,7 +1,7 @@
 import type { GlobalConfigs, ManifestOptions, ResourceTypeConstraint } from './manifest'
 export type { GlobalConfigs, ManifestOptions, ResourceTypeConstraint }
 export type { Request } from './request'
-export type { Headers, Params as Parameters } from './types'
+export type { Headers, Params as Parameters, Auth as Authorization } from './types'
```

Comment:
`Auth` from `types` and `Authorization` from `middleware` are a tidbit different, in that `Auth` has a more generic index signature. Are we fine with that?

Reply:
Yes, `Authorization` from middleware is deleted and internally it is using from `types` as well. It is non-breaking too since it went from strict to laxer type.